helm-charts/codegen/templates/deployment.yaml

```diff
@@ -32,6 +32,12 @@ spec:
           env:
             - name: LLM_SERVICE_HOST_IP
               value: {{ .Release.Name }}-llm-uservice
+            - name: HTTP_PROXY
+              value: {{ .Values.global.http_proxy }}
+            - name: HTTPS_PROXY
+              value: {{ .Values.global.https_proxy }}
+            - name: NO_PROXY
+              value: {{ .Values.global.no_proxy }}
```

Comment:
I'm wondering if the mega service itself needs proxy settings?

Reply:
I agree with lianhao. And also, if we need proxy settings, shall we do the same to chatqna?

Reply:
I think basically, we need to populate proxy settings to any service which needs to talk to internet, i.e. services with HF API token to be verified, services with model to be downloaded from HF, etc.

Reply:
Remove proxy setting for mega service, keep it for llm-uservice and tgi which need to talk to outside.